src/components/Datepicker/test/datepicker.test.js

```diff
@@ -1,10 +1,67 @@
 import React from "react";
 import DatePicker from "../Datepicker";
 import { create } from "react-test-renderer";
+import { render } from "@testing-library/react";
 
 describe("DatePicker component", () => {
   test("Matches the snapshot", () => {
-    const spinner = create(<DatePicker value={{ year: 2020, month: 12 }} />);
-    expect(spinner.toJSON()).toMatchSnapshot();
+    const datePicker = create(<DatePicker value={{ year: 2020, month: 12 }} />);
+    expect(datePicker.toJSON()).toMatchSnapshot();
+  });
+
+  test("lang prop values are used as months", () => {
+    const years = {
+      min: { year: 2018, month: 1, day: 25 },
+      max: { year: 2020, month: 2, day: 4 }
+    };
+    const { queryByText, rerender, unmount } = render(
+      <DatePicker
+        lang={[
```

Comment:
can you create a variable for this lang array?

Reply:
:100: